extensions/extensions-helm/extensions-helm-pipeline-invocation/aissemble-pipeline-invocation-app-chart/values.yaml

```diff
@@ -12,7 +12,7 @@ aissemble-quarkus-chart:
         containerPort: 9000
         protocol: TCP
     restartPolicy: Always
-    serviceAccountName: sparkoperator
+    serviceAccountName: pipeline-invocation-service
```

Comment:
A: Default name of `pipeline-invocation-service-account` or `pipeline-invocation-sa` would be better. Calling it service can easily get mixed up with the kubernetes service.